package.json

```diff
@@ -9,6 +9,7 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
+	"funding": "https://github.com/sindresorhus/ky?sponsor=1",
```

Comment:
reminder, this won't work at the moment ☺️

Reply:
Yes, I'll merge this PR when it's fixed. I don't want to keep changing the format, so I'd rather wait.